src/Microsoft.IdentityModel.Clients.ActiveDirectory/AdalError.cs

```diff
@@ -0,0 +1,280 @@
+﻿//----------------------------------------------------------------------
```

Comment:
Is it related to the breaking change?

Reply:
No, but chris said that this file was being reported as internal namespace. not sure why, so I moved out the public static class to its own file for safety